benchmarks/buiter_et_al_2016_jsg/README

```diff
@@ -0,0 +1,40 @@
+This benchmark attempts to reproduce the results of Buiter et al. 2016:
+  Benchmarking numerical models of brittle thrust wedges,
+  Journal of Structural Geology, 92, 140-177, 
+  http://dx.doi.org/10.1016/j.jsg.2016.03.003.
+
+More specifically, the provided input files aim to reproduce the numerical simulations 
+of stable wedge experiment 1 illustrated in Figure 4-6 and unstable wedge experiment 2 
+illustrated in Figure 8-11. The setups of these two experiments are shown in Figure 3a-b. 
+
+Experiment 1 tests whether model wedges in the stable domain of critical wedge theory 
+remain stable when translated horizontally. A quartz sand wedge with a horizontal base 
+and a surface slope of 20 degrees is pushed horizontally by inward movement of a mobile 
+wall at the right boundary with a velocity of 2.5 cm/hour.
+  
+[...]
```

Comment:
Do you intend to add more text here? Or are you actually quoting from the paper?

Reply:
If this is *your* text, I think it would be useful to make clear when you reference a figure for the first time that you are referencing a figure *in the Buiter et al. paper*.

Reply:
> Do you intend to add more text here? Or are you actually quoting from the paper?

This part is left for the description of experiment 1, where Stephanie will add more text after merging this benchmark.

 
> If this is _your_ text, I think it would be useful to make clear when you reference a figure for the first time that you are referencing a figure _in the Buiter et al. paper_.

Good Point. Thanks!